src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -610,6 +617,20 @@ const PIAFormPage = () => {
           status: PiaStatuses.FINAL_REVIEW,
         });
 
+        if (updatedPia?.id) {
+          navigate(
+            buildDynamicPath(routes.PIA_VIEW, {
+              id: updatedPia.id,
+            }),
+          );
+        }
+      } else if (buttonValue === 'completePIA') {
+        const updatedPia = await upsertAndUpdatePia({
+          // here not sure what status for this one, need to discuss
```

Comment:
do you need discussion here?

Reply:
confirm, right now route to pia intake tab